src/translations/en.json

```diff
@@ -1462,7 +1462,8 @@
             "import_btn": "Import blueprint",
             "saving": "Saving blueprint...",
             "save_btn": "Save blueprint",
-            "error_no_url": "Please enter the url of the blueprint."
+            "error_no_url": "Please enter the url of the blueprint.",
```

Comment:
Can you make that "URL" to be consistent with the other messages?

Reply:
```suggestion
            "error_no_url": "Please enter the URL of the blueprint.",
```